### PR TITLE
Added missing deviation and resource name for Nokia

### PR DESCRIPTION
**Pull request by bkreddy143**
1. Added deviations required for Nokia
2. update Nokia Fib resource  name for Nokia 
3. As per README there is no requirement for active-component-list support  so modified logic to get active intergrated-circuit list from components/component tree.  Please refer b/281931161 ( comment#43) for more details on the same . 
4.  Added common function to get active components by type .

"This code is a Contribution to the OpenConfig Feature Profiles project ("Work") made under the Google Software Grant and Corporate Contributor License Agreement ("CLA") and governed by the Apache License 2.0. No other rights or licenses in or to any of Nokia's intellectual property are granted for any other purpose. This code is provided on an "as is" basis without any warranties of any kind."

### CODE REVIEW

File: feature/platform/integrated_circuit/otg_tests/utilization_test/utilization_test.go [hunk 1]
```diff
@@ -105,36 +107,12 @@ func TestResourceUtilization(t *testing.T) {
...
-	}).Await(t)
-	if !ok {
-		t.Fatalf("FIB resource is not active in any available components")
-	}
-	comps, _ = val.Val()
-
+	comps := components.FindActiveComponentsByType(t, dut, oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_INTEGRATED_CIRCUIT)
 	beforeUtzs := componentUtilizations(t, dut, comps)
 	if len(beforeUtzs) != len(comps) {
 		t.Fatalf("Couldn't retrieve Utilization information for all Components in active-component-list")
```
Comment (cfernanz) [inline feature/platform/integrated_circuit/otg_tests/utilization_test/utilization_test.go:118]: Remove "active-component-list" from error message.

Same for line 125

File: internal/components/components.go [hunk 2]
```diff
@@ -58,6 +58,28 @@ func FindComponentsByType(t *testing.T, dut *ondatra.DUTDevice, cType oc.E_Platf
 	return s
 }
 
+// FindComponentsByType finds the list of components based on hardware type.
```
Comment (cfernanz) [inline internal/components/components.go:61]: Replace with FindActiveComponentsByType

File: feature/platform/integrated_circuit/otg_tests/utilization_test/utilization_test.go (hunk 1 above)
Comment (bkreddy143) [reply feature/platform/integrated_circuit/otg_tests/utilization_test/utilization_test.go]: Updated , thanks !

File: internal/components/components.go (hunk 2 above)
Comment (bkreddy143) [reply internal/components/components.go]: updated, thanks !

**Review by nsadhasivam — APPROVED** [review]
LGTM!

**Pull request merged**